specification/under_development/output_schemas/Labeled_And_Async.md

```diff
@@ -1,2 +1,238 @@
 # Labeled Format for Asynchronous Output Generation
```

Comment:
Nitpick: throughout the document labels and tags are used interchangeably, maybe we could consistently just use labels?